### PR TITLE
fix(dataManager): fix clear form validate

### PR DESCRIPTION
**Pull request by ysfscream**

### CODE REVIEW

File: src/views/Auth/components/DataManager.vue [hunk 1]
```diff
@@ -70,17 +71,19 @@
       :title="isEdit ? $t('Base.edit') : $t('Base.add')"
       width="480px"
       v-model="dialogVisible"
+      @open="handleDialogOpen"
     >
       <el-form ref="recordForm" :model="record" :rules="getRules()" label-position="top">
         <el-form-item prop="user_id" :label="getFiledLabel(field)">
-          <el-input v-model="record.user_id" :disabled="isEdit"></el-input>
+          <el-input size="small" v-model="record.user_id" :disabled="isEdit"></el-input>
```
Comment (Kinplemelon) [inline src/views/Auth/components/DataManager.vue:78]: Can this property be set on the form component?

File: src/views/Auth/components/DataManager.vue [hunk 2]
```diff
@@ -283,8 +294,33 @@ export default defineComponent({
       }
       return fieldMap[field]
     }
+    const handleDialogOpen = () => {
+      recordForm.value?.clearValidate()
+      recordForm.value?.resetFields()
+    }
+    const handleSearch = () => {
+      const page = 1
+      console.log(searchVal)
```
Comment (Kinplemelon) [inline src/views/Auth/components/DataManager.vue:303]: for test?

Comment (ysfscream) [reply src/views/Auth/components/DataManager.vue:303]: sorry! We should delete it.

File: src/views/Auth/components/DataManager.vue (hunk 1 above)
Comment (ysfscream) [reply src/views/Auth/components/DataManager.vue:78]: Good suggestion

Comment (ysfscream) [reply src/views/Auth/components/DataManager.vue:78]: But the size of the UI components, we need to discuss and keep consistent during the redesign

Comment (Kinplemelon) [reply src/views/Auth/components/DataManager.vue:78]: 😲

**Pull request merged**